.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: python
 python:
   - 3.5
+services:
+  - postgresql
 env:
   - TRAVIS_NODE_VERSION="4.4"
 install:
   - pip install -r requirements.txt
-  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
-  - npm install
```

Comment:
we'll need to add this back soon (forgot to update our README), as this is used for sass/css compilation, but fine to leave it out for now